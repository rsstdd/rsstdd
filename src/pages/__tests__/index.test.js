import * as React from "react"
import { render } from "@testing-library/react"

import IndexPage from "../index"

describe("IndexPage suite", () => {
  it("IndexPage should render", () => {
    const wrapper = render(<IndexPage />)

    expect(wrapper.container).toMatchSnapshot()
  })
})
