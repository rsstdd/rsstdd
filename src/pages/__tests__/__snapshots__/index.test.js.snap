// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage suite IndexPage should render 1`] = `
<div>
  <h1>
    Hi people
  </h1>
  <p>
    Welcome to your new Gatsby site.
  </p>
  <p>
    Now go build something great.
  </p>
  <a
    href="/page-2/"
  >
    Go to page 2
  </a>
</div>
`;
